doorpi.py

```diff
@@ -294,6 +297,12 @@ def send_update(cls, message):
             except:
                 logging.error("Error sending message", exc_info=True)
 
+class Door:
+
+    # class constructor
+    def __init__(self, name, gpioOpenPin):
```

Comment:
Door.gpioOpenPin is IMO miss-leading because above you are passing a GPIO.DigitalOutputDevice ... passing the pin and instantiating the DigitalOutputDevice could make more sense.